ts/utils/dates.ts

```diff
@@ -23,12 +23,26 @@ export function formatDateAsReminder(
   return dateFnsFormat(date, "YYYY-MM-DDTHH:mm:ss.SSS[Z]");
 }
 
+export function formatDateAsLocal(
+  date: Date,
+  includeYear: boolean = false
+): ReturnType<typeof dateFnsFormat> {
+  enum LocaleFormat {
+    "" = "MM/DD",
+    "en" = "MM/DD",
```

Comment:
these formats should go [here](https://github.com/teamdigitale/io-app/blob/master/locales/it/index.yml#L5)